cylc/flow/task_remote_mgr.py

```diff
@@ -344,8 +344,9 @@ def remote_tidy(self) -> None:
         Timeout any incomplete commands after 10 seconds.
         """
         # Get a list of all platforms used from workflow database:
-        platforms_used = (
-            self.db_mgr.get_pri_dao().select_task_job_platforms())
+        with self.db_mgr.get_pri_dao() as pri_dao:
+            platforms_used = pri_dao.select_task_job_platforms()
```

Comment:
Ensure the DB gets closed.

Reply:
Hmm, I suggested this twice in my review of #5445, both times it was marked as resolved but never went in!

Reply:
I'm going to just mark this as resolved :see_no_evil: